.pipelines/cni/pipeline.yaml

```diff
@@ -153,6 +169,34 @@ stages:
       vmSize: ${VM_SIZE}
       arch: amd64
 
+  - template: singletenancy/linux-cniv2-template.yaml
+    parameters:
+      name: mariner_linux_overlay
+      clusterType: overlay-byocni-up
+      clusterName: mariner-cniv2-overlay
+      nodeCount: ${LINUX_NODE_COUNT}

```

Comment:
I think this will change in accordance with the PR by @jpayne3506: https://github.com/Azure/azure-container-networking/commit/f0021b58e865ce9a30bf64adb02cdddc1f8cb82b#diff-345f445e69479edda267b386cd089556500ef27605c05b5a9d16c4ae0ca61b29R131-R132